src/app/_components/SearchInput.tsx

```diff
@@ -1,20 +1,23 @@
 import { ChangeEvent } from 'react'
 
 import { MagnifyingGlass } from '@phosphor-icons/react/dist/ssr'
+import clsx from 'clsx'
 
 import { Icon } from '@/components/Icon'
 
-interface SearchInputProps {
+export type SearchInputProps = {
   searchQuery: string
   onSearchChange: (event: ChangeEvent<HTMLInputElement>) => void
+  width?: 'full' | 'xs'
 }
 
 export function SearchInput({
   searchQuery = '',
   onSearchChange,
+  width = 'xs',
 }: SearchInputProps) {
   return (
-    <div className="w-full max-w-lg lg:max-w-xs">
+    <div className={clsx('w-full', width === 'xs' && 'max-w-lg lg:max-w-xs')}>
```

Comment:
To me, this feels like "reaching into the component" - Layout styles usually can be handled from the parent. I'm going to remove this for now. It will still be available as a solution in the PR history, so we can always pull it back if needed - though I think it might not be.

Reply:
In that case, we need to remove `max-w-lg lg:max-w-xs` from this component so that it takes the full width of the parent's container.